### ISSUE TITLE
Applying custom msstyles in Windows 11 (maybe) doesn't work

### ISSUE THREAD

**Issue by Hibiki0506** [nptr/msstyleEditor#81]
*Windows Version*: Win11 22H2 Insider (Build 10.0.22610.1).>>

**Expected behavior**
I was using a copied aero.msstyles from Windows 11 and wanted to apply it without changing anything to the same system, which should work.

**Actual behavior**
When I was loading the msstyles from my System, msstyleEditor 2.0.0.0 was giving me the information, that this file was not made for this version of Windows, which is weird,, since it was the msstyles of my system. When I wanted to apply the msstyles onto my system, I've got the error message shown in the screenshot at **Additional context**.

**Additional context**
![image](https://user-images.githubusercontent.com/32351488/167081900-9835f983-b659-4d51-950d-859daa2e4835.png)
(Translation for the few German words ("Element nicht gefunden"): "Object not found"
**Checklist**
- [x] My uxtheme.dll is patched
- [x] I am using the latest version of msstyleEditor
- [x] None of the other issues cover this

**Comment by Hibiki0506** [comment]
Yes, here is the Aero folder. But sadly, I can't copy the VSCache folder in the Aero folder, since I don't even have the right to look into the folder without breaking any stuff.

I hope that there isn't anything important in the VSCache folder.

[aero.zip](https://github.com/nptr/msstyleEditor/files/8641292/aero.zip)

**Comment by nptr** [comment]
Thanks!

* I fixed the arguably less important issue of incorrectly detecting Win11. Silly me assumed Microsoft would increment dwMajorVersion from 10 to 11 upon release.... I'll release a fixed version once I fixed the other bug.

* Not sure what exactly causes the issue with not being able to apply the theme. Funnily my Win10 can apply it, so I'll have to spin up a Win11 VM somewhere in the future and properly debug. I assume one of the PE resources can't be located because of language dependency issues. 

  If you are motivated however and want to speed things up, you could see if msstyleEditor can open and test one or more of the attached styles. Those are from your aero style, but I modified some resources' language IDs to see if its still the same issue I attempted to fix recently.

Aero styles to test: [w11_aero_tests.zip](https://github.com/nptr/msstyleEditor/files/8643827/w11_aero_tests.zip)

**Comment by Hibiki0506** [comment]
Hmm, I can apply them all on my installation of Windows 11, so whatever you did, it is working, somehow. But since it is the original msstyles from Windows 11 22H2, I can't really tell if they are applied correctly. But gladly there isn't an error anymore, so I could try to edit the theme to see if there is something what I can see.

**Comment by Hibiki0506** [comment]
After testing to replace some graphics with their Windows 8.1 counterparts, there seems to nothing is changed by replacing them in your edited Windows 11 msstyles. I've tested it with the scroll bars, since they have very different looks by now with the sharp corners in Win8.1 and the rounded ones from Windows 11, but applying the msstyles, nothing changed, even after resetting explorer.exe.

**Comment by nptr** [comment]
I usually export, paint over and import the pushbutton or checkbox images to verify the style is applied. Does this show any effect?

**Comment by Hibiki0506** [comment]
No, nothing changed. It's still the same.

**Comment by nptr** [comment]
Would you also attach the style you modified (with the Windows 8.1 images)? Maybe saving the style goes wrong now.

**Comment by Hibiki0506** [comment]
Hey, sorry that I couldn't replay earlier.
Here are the edited msstyles, one with MUI saved and one without, maybe it can help you.


[Win11Test.zip](https://github.com/nptr/msstyleEditor/files/8649751/Win11Test.zip)

**Comment by Hibiki0506** [comment]
I am sure that I was using the right files for 100% DPI settings @ 1920x1080 screen resolution. 

But I am unsure, if this should work, since I was just replacing the same files as before in Windows 8.1 for the scrollbars and I am looking in the explorer windows, which I am opening after applying the theme.

**Comment by nptr** [comment]
The following all work fine for me:
- `DarkMode_Explorer::ScrollBar` (the Explorers scrollbar in dark mode)
- `ScrollBar` (the non-Explorer scroll bar for regular mode)

I tested on the `GRIPPERHORZ` and `ARROWBTN `and `THUMBBTNHORZ `parts.

I also made [a new version](https://github.com/nptr/msstyleEditor/files/8664440/msstyleEditor2001.zip), it doesn't really include any fixes specific to the issue you are reporting because I can not reproduce it. I'd like to share it nonetheless.
If the problem you encountered persists, I'll need to gather more insight.
